vetiver/handlers/sklearn.py

```diff
@@ -15,24 +14,14 @@ class SKLearnHandler(BaseHandler):
     """
 
     model_class = staticmethod(lambda: sklearn.base.BaseEstimator)
+    pip_name = "scikit-learn"
```

Comment:
is there a way to get these values without manually setting them? my concern is that some packages (scikit-learn is one of them) has a different name on pip vs when you are importing it.

Reply:
I don't think there's another way to get it, since you could upload the same package to several pypi repo names if you wanted to :/. (but could be wrong)

Reply:
> I don't think there's another way to get it, since you could upload the same package to several pypi repo names if you wanted to :/. (but could be wrong)

A case of "packages are not first class objects" in python.